lib/galaxy/managers/hdcas.py

```diff
@@ -335,3 +337,12 @@ def serialize_job_state_summary(self, item, key, **context):
     def serialize_elements_datatypes(self, item, key, **context):
         extensions_set = item.dataset_dbkeys_and_extensions_summary[1]
         return list(extensions_set)
+
+
+def get_hdca_by_name(session, name):
```

Comment:
This doesn't seem to belong in managers to me because it doesn't use trans or app. Also this code is only used in tests as far as I can grep - should just be moved to ``galaxy.model.unittest_utils.model_testing_utils`` or a new file like ``galaxy.model.unittest_utils.helpers``. I think there is something jarring about seeing code like this mixed with app logic - I was immediately scared we were accessing data this way from the app.